tests/director/tests/application_templates_subscription_test.go

```diff
@@ -24,87 +24,89 @@ import (
 
 func TestSubscriptionApplicationTemplateFlow(stdT *testing.T) {
 	t := testingx.NewT(stdT)
-	ctx := context.Background()
-
-	// GIVEN
-	subscriptionProviderSubaccountID := conf.TestProviderSubaccountID
-	subscriptionConsumerSubaccountID := conf.TestConsumerSubaccountID
-	subscriptionConsumerTenantID := conf.TestConsumerTenantID
-
-	// Prepare provider external client certificate and secret and Build graphql director client configured with certificate
-	providerClientKey, providerRawCertChain := certprovider.NewExternalCertFromConfig(stdT, ctx, conf.ExternalCertProviderConfig)
-	directorCertSecuredClient := gql.NewCertAuthorizedGraphQLClientWithCustomURL(conf.DirectorExternalCertSecuredURL, providerClientKey, providerRawCertChain, conf.SkipSSLValidation)
-
-	apiPath := fmt.Sprintf("/saas-manager/v1/application/tenants/%s/subscriptions", subscriptionConsumerTenantID)
-	subscriptionToken := token.GetClientCredentialsToken(stdT, ctx, conf.SubscriptionConfig.TokenURL+conf.TokenPath, conf.SubscriptionConfig.ClientID, conf.SubscriptionConfig.ClientSecret, "tenantFetcherClaims")
+	t.Run("Success", func(t *testing.T) {
```

Comment:
It has to be not a common name here because other tests that have skip patterns could match and skip them unintentionally. Also, change the regex pattern in our internal repo as well

Reply:
Why unique? If someone would create a regex for his test, he'd probably use a pattern like `MyRootTest_Success` or something like that.
Also the general regex in the internal repo would catch the whole test suite - why changing it?

Reply:
About the first thing, it's hard to predict what someone will do or will not do. And creating it with a common name we risk skipping other tests which I think is not what we want, also, it will be very hard for us to notice some tests are skipped. However, the "skip mechanism" that we have just does not work that way. It aims to match and skip a "test run" not a "test suite" you're creating and the name of the test suite should not be included in the skip regex pattern. So if you do not change the regex in our internal repo I think the result will be the same and that change will be obsolete. You could drill down to double-check that.

Reply:
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/17293744/176618397-8e35deca-f017-4717-81b7-603af865ff15.png">
A local test run with a hardcoded pattern. Seems like the test suite is skipped after all.
I can change the sub-test name, tho.